homeassistant/components/tedee/entity.py

```diff
@@ -32,6 +32,7 @@ def __init__(
             name=lock.lock_name,
             manufacturer="Tedee",
             model=lock.lock_type,
+            model_id=lock.lock_type,
```

Comment:
These are now the same?

It this the model ID they put on their labeling on the device as well?

Reply:
yes, afaik there are no other identifiers for the models